application-ideas-ui/src/main/resources/Ideas/IdeasClass.xml

```diff
@@ -102,7 +102,7 @@
     </content1>
     <nbagainst>
       <customDisplay>{{velocity}}
-  $xwiki.jsx.use("Ideas.IdeasClass")
+  $xwiki.jsx.use("Ideas.IdeasClass", {'docRef': $doc.documentReference})
```

Comment:
Why do you need to pass the document reference? You have access to the current document reference from JavaScript.

Reply:
The document reference was needed because I wanted to use `$services.ideas.url($request.docRef)`. I won't need that anymore now.